backend/molgenis-emx2-semantics/src/main/java/org/molgenis/emx2/semantics/LinkedDataService.java

```diff
@@ -79,6 +80,13 @@ public static void getJsonLdForTable(Table table, PrintWriter writer) {
           }
           // add to select
           q.select(s(c.getName(), s(refId.get(0).getName())));
+        } else if (c.isReference() && c.isOntology()) {
```

Comment:
probably only need c.isOntology (which implies c.isReference, right?)

I don't require you change, then resolve without action.

Reply:
Fixed